src/html/index.html

```diff
@@ -13,8 +13,9 @@
     </head>
     <body>
         <!-- Content -->
-        <h1>Practice Project</h1>
-        <p>Practice project for HTH</p>
+        
+        <center><h1>Practice Project</h1>
```

Comment:
I would prefer the center tags on separate lines and the contents indented for consistency.